### DISCUSSION TITLE
The account was stolen

### DISCUSSION THREAD

**Discussion by polikhronidi1** [community/community#128971] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello guys, I am writing to inform you that my GitHub account has been compromised. The unauthorized individual(s) have changed both the email address associated with the account and the password, rendering me unable to access it. The email and password were changed, so I can't access my account even with "forgot password" and 2FA enabled. Please help me recover it.

Here are the details of my account:

Username: github.com/polikhronidi
Original Email: pn198047f*****3@gmail.com

I believe my account's security has been breached, and I kindly request your immediate assistance in restoring access to my account. Please let me know what steps I should take to verify my identity and regain control of my GitHub account.

![image](https://github.com/community/community/assets/173175862/f9f7609d-2275-4dda-bde5-2f4682cfca4f)

**Comment by ebndev** [comment]
Hi @polikhronidi1,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
